src/ui/components/layouts/base_layout/side_nav.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+  ui::components::common::icon::{Icon, IconType},
+  use_i18n,
+};
+use leptos::*;
+use leptos_i18n::t;
+use leptos_router::A;
+
+#[component]
+pub fn SideNav() -> impl IntoView {
+  let i18n = use_i18n();
+
+  view! {
+    <nav aria-label="Pages nav">
+      <ul>
+        <NavLink href="/create_post" icon=IconType::CreatePost text=t!(i18n, create_post)/>
+        <NavLink
+          href="/create_community"
+          icon=IconType::CreateCommunity
+          text=t!(i18n, create_community)
+        />
+        <NavLink href="/communities" icon=IconType::Communities text=t!(i18n, communities)/>
+        <NavLink href="/search" icon=IconType::Search text=t!(i18n, search)/>
+        <NavLink href="/modlog" icon=IconType::Modlog text=t!(i18n, modlog)/>
+        <NavLink href="/instances" icon=IconType::Instances text=t!(i18n, instances)/>
+        <NavLink href="/legal" icon=IconType::Legal text=t!(i18n, legal)/>
+      </ul>
+    </nav>
+    <nav aria-labelledby="lemmy-resources-label" class="mt-auto">
+      <div id="lemmy-resources-label" class="font-medium mb-1">
+        Lemmy Resources
+      </div>
+      <ul>
+        <NavLink
```

Comment:
All these links might be used elsewhere, so it'd be good to externalize them to a constants / common file.